tests/middleware/tests.py

```diff
@@ -66,6 +66,15 @@ def test_append_slash_redirect(self):
         self.assertEqual(r.status_code, 301)
         self.assertEqual(r.url, '/slash/')
 
+    @override_settings(APPEND_SLASH=True)
+    def test_append_slash_redirect_querystring(self):
+        """
+        Tests that APPEND_SLASH preserves querystrings when redirecting.
```

Comment:
prefer simply describing the behavior: "APPEND_SLASH should preserve ..." rather than prefixing every docstring with "Tests that"